### ISSUE TITLE
Unable to write test script 

### ISSUE THREAD

**Issue by tcho16** [Mashape/kong#1982]
### Summary

I am trying to test a simple test file so I can test my plugin using busted. When I run `kong start -c /usr/local/share/lua/5.1/kong/kong_TEST.yml`, kong starts perfectly and if I go to http://localhost:8101/apis, it shows the number of apis I currently have. I also have busted installed and when I go to the folder containing the test (access_spec.lua) and run `busted access_spec.lua`, it gives me the following error: `[ERR] No configuration file at: kong/kong_TEST.yml`. What I've also noticed is that if I go to a random directory and type in busted, I get the usual message `No test found matching the pattern _spec.lua` but when I go to `/usr/local/share/lua/5.1/kong/kong_TEST.yml` it gives the error `[ERR] No configuration file at: kong/kong_TEST.yml`. My test file looks like [this](http://streamdata.io/blog/developing-an-helloworld-kong-plugin/).

### Steps To Reproduce

1.  Start kong using the above kong start command
2. Navigate to the test file and run the busted command

### Additional Details & Logs

- Kong version (`$0.7.0`)
- centOS 7

**Comment by Tieske** [comment]
not sure for 0.7.0 anymore, thats an older version as were already releasing 0.10.0 these days. But personally I test from the repo root using something like

```
./bin/busted ./spec/02-integration/etc_spec.lua
```

**Comment by tcho16** [comment]
Okay so I've managed to fix that issue. The fix behind that was to explicitly write the file path in the spec file on the _M.TEST_CONF_FILE variable. But now I'm still unable to run the test. In the test, there is a start method that starts up kong but I get an error that it can't start up kong because kong is already running. But I don't see how that's possible since I made sure kong was not running prior to running the test by running `kong stop`.

 I've also noticed that even after restarting my system, in the root user, kong is still running. Surely it shouldn't be running unless I explicity write 'kong start' etc..
